neo/Network/RPC/RpcClient.cs

```diff
@@ -278,6 +286,14 @@ public bool SendRawTransaction(byte[] rawTransaction)
             return RpcSend("sendrawtransaction", rawTransaction.ToHexString()).AsBoolean();
```

Comment:
Maybe add serialized to the previous description: "Broadcast a serialized transaction over ..."